fix: support for iOS 14+ with `setAdvertiserTrackingEnabled`; remove rectangle banner on iOS (#280)
callstack/react-native-fbads: README.md

- `standard` (BANNER_HEIGHT_50)
- `large` (BANNER_HEIGHT_90)

> - `rectangle` (RECTANGLE_HEIGHT_250: (currently only for Android)

Reviewer: ```suggestion
- `rectangle` (RECTANGLE_HEIGHT_250) – currently Android only
```

```js
const trackingStatus = await AdSettings.requestTrackingPermission();
if (trackingStatus === 'authorized' || trackingStatus === 'unavailable') {
  AdSettings.setAdvertiserIDCollectionEnabled(true);

>   AdSettings.setAdvertiserTrackingEnabled(true);

Reviewer: this is unrelated, let's remove

}
```

> Requires iOS 14. On Android and iOS versions below 14, this will always be no-op.

> ### setAdvertiserTrackingEnabled

Reviewer: Let's add a clear notice that FB won't display adds when this is not set to true

> Enables or disables ads tracking. Since the iOS 14 API has been introduced you will need set this flag in order to inform Facebook whether you should receive personalized ads. ( See [Advertising Tracking Enabled For Audience Network](https://developers.facebook.com/docs/audience-network/guides/advertising-tracking-enabled))

Reviewer: ```suggestion
Enables or disables personalized ads tracking on iOS 14+. See Facebook docs on [Advertising Tracking Enabled For Audience Network](https://developers.facebook.com/docs/audience-network/guides/advertising-tracking-enabled)
```

}
```

### setAdvertiserTrackingEnabled

Enables or disables personalized ads tracking on iOS 14+. See Facebook docs on [Advertising Tracking Enabled For Audience Network](https://developers.facebook.com/docs/audience-network/guides/advertising-tracking-enabled)

> Requires iOS 14. On Android and iOS versions below 14, this will always be no-op.

> > **Note**: FB won't display adds when this is not set to true.

Reviewer: ```suggestion
> **Important: FB won't display adds until this is set to `true`.**
```
